src/sgame/sg_admin.cpp

```diff
@@ -6151,16 +6151,39 @@ bool G_admin_bot( gentity_t *ent )
 
 bool G_admin_listbots( gentity_t *ent )
 {
+	struct description_t
```

Comment:
I don't see the motivation for defining a struct. Just make a vector of client nums or `gentity_t *`s and sort that.

Reply:
motivation was to hack it quickly, and I didn't thought about using gentity_t. Will change that.

Reply:
well, to be honest, I'm wondering if it's worth it at all. I still have to see any real benefit the skillset feature brings. The problems it brings, those I already know.
Would this change be required? Because this patch is self-constrained, that struct is not usable out of this scope... and I'm not sure I want to spend that much time on a feature that is mostly not maintained so little time after it's integration.

Reply:
Yeah I'm requiring it, the struct really makes the code significantly more complex to read. Sure, you can remove the team sorting

Reply:
removed.